### ISSUE TITLE
Unable to create new TrackingOptions

### ISSUE THREAD

**Issue by willpower232** [calcinai/xero-php#871]
I would like to create a TrackingOption as part of preparing a Demo Company for development.

Unfortunately the following code does nothing

```php
$option = new TrackingOption($xeroConnection);
$option->setName($email);
$trackingCategory->addOption($option);
$trackingCategory->save();
```

Having inspected the savePropertiesDirectly method, I extracted the following XML body

```xml
<s><s><Name>email@example.com</Name></s></s>
```

I can see that in #801 @philipdarlo set `TrackingOption::getRootNodeName` to an empty string for some reason.

I can obviously PR to restore the word "Option" but was there any particular reason for blanking the return?

**Comment by willpower232** [comment]
a quick solution is to use your own class when creating the option but that doesn't help you if you want to edit them

```php
<?php

namespace App\Xero\Models;

use XeroPHP\Models\Accounting\TrackingCategory\TrackingOption as ParentTrackingOption;

class TrackingOption extends ParentTrackingOption
{
    /**
     * @inheritdoc
     */
    public static function getRootNodeName()
    {
        return 'Option';
    }
}
```

**Comment by calcinai** [comment]
@willpower232does this work? Over the years there has been inconstant behaviour handling Tracking Categories and options.

**Comment by willpower232** [comment]
I can definitely create tracking options now and the XML is clearly broken without it so I'd say this is definitely needed.

Unless there was some wildly different intention from that PR, I'm not sure how they're possibly using the model although the addition of methods in `getSupportedMethods` implies they're doing something weird since tracking options don't get saved directly https://developer.xero.com/documentation/api/accounting/trackingcategories

**Comment by calcinai** [comment]
Yeah maybe something has changed, or we've finally found a solution? Let's merge this because, as you say, it's not working as it is.

**Comment by philipdarlo** [comment]
Apologies, for not getting a reply to the tag above on this sooner. 

When I created the original MR, I was taking someones older work and just making sure it had no conflicts so could be merged.

I ended up not using this feature in my project and instead doing a manual work around.

I apologies @willpower232 as I did not test this well enough as has been clearly shown in this issue.
